code/mathematical_algorithms/src/freivalds_algorithm .cpp

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
The location of this file is incorrect. Also, your file extension seems detached from the name.